### ISSUE TITLE
exclude stuff

### ISSUE THREAD

**Issue by stephanenicolas** [kt3k/coveralls-gradle-plugin#62]
It looks like jacoco excludes are not taken into account by coveralls and it has its own python based exclude mechanism. It would be great to be able to configure excludes. My coverage is dropping by 10 percent for a class that I had to copy into my own project, and this class is already tested in the original project it belongs to. 

And no, I can 't copy the tests, they use different testing lib..

**Comment by kt3k** [comment]
I guess you can add `excludes` by configuring `JacocoTaskExtension`. See the jacoco plugin document, https://docs.gradle.org/current/userguide/jacoco_plugin.html#N16C9E .

**Comment by stephanenicolas** [comment]
yes but coveralls doesn't take it into account. Jacoco seems to only
exclude from its own reports.

2016-05-01 19:05 GMT-07:00 Yoshiya Hinosawa <notifications@github.com>:
